worker/raft/raftforwarder/worker.go

```diff
@@ -129,15 +141,18 @@ func (w *forwarder) handleRequest(_ string, req raftlease.ForwardRequest, err er
 			w.catacomb.Kill(errors.Annotate(err, "publishing response"))
 			return
 		}
+		w.metrics.record(start, "full")
 	}()
 }
 
 func (w *forwarder) processRequest(command string) (raftlease.ForwardResponse, error) {
 	var empty raftlease.ForwardResponse
+	start := time.Now()
 	future := w.config.Raft.Apply([]byte(command), applyTimeout)
 	if err := future.Error(); err != nil {
 		return empty, errors.Trace(err)
```

Comment:
Wouldn't it make sense to have in a record before the return as well?

Reply:
I guess there is a potential here that future.Error is returning an error sometimes

Reply:
I wasn't going to include errors in the stats - do you think I should, as another axis?

Reply:
I think it would be interesting to see an error rate for the future. It should be really low right?